docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire project directory as a volume (`./:/app`) is suitable for development but not recommended for production due to potential security risks and performance issues. Consider using Docker's build context and COPY instructions for production.